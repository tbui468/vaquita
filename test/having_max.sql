if exists drop database sol;
create database sol;
open sol;

create table planets (
                        name string, 
                        mass float, 
                        moons int,
                        ring_system bool
                    );

insert into planets (
                        name, 
                        mass, 
                        moons,
                        ring_system
                    ) 

values
        ("Venus", 2.32, 0, false),
        ("Earth", 5.0, 1, true),
        ("Mars", 4.23, 2, true),
        ("Saturn", 3.23, 2, true);

select ring_system, max(mass) from planets group by ring_system having max(mass) > 0.0 and max(mass) < 4.0;
select ring_system, max(mass) from planets group by ring_system having max(mass) > 0.0 and max(mass) < 6.0;
close sol;
drop database sol;
exit;
