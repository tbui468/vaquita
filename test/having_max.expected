created database sol
opened database sol
created table planets
inserted 4 record(s) into planets
false, 2.320000, 
false, 2.320000, 
true, 5.000000, 
closed database sol
dropped database sol
